{"name": "test_c002_04_error_field", "status": "passed", "steps": [{"name": "Signup unsuccessful, error field validation", "status": "passed", "start": 1631722850948, "stop": 1631722850948}], "start": 1631722850948, "stop": 1631722850948, "uuid": "94d87a60-2b12-4a8c-87b7-c9b85985ba9d", "historyId": "94de3f2bf73ace946eb1f57f28927957", "testCaseId": "69f59686174a6a09577bfc334f58549e", "fullName": "testcases.test_c002_signup_400#test_c002_04_error_field", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_c002_signup_400"}, {"name": "host", "value": "QupsAnik"}, {"name": "thread", "value": "13468-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_c002_signup_400"}]}